docs/3.0.0-beta.x/guides/graphql.md

```diff
@@ -774,7 +774,7 @@ You can also execute a custom logic like above. However, the roles and permissio
 
 ### Apply permissions on a query
 
-It might happens that you want apply our permissions layer on a query. That's why, we created the `resolverOf` attribute. This attribute defines which are the permissions that should be applied to this resolver. By targeting an action it means that you're able to edit permissions for this resolver directly from the administration panel.
+It might happen that you want apply our permissions layer on a query. That's why, we created the `resolverOf` attribute. This attribute defines which are the permissions that should be applied to this resolver. By targeting an action it means that you're able to edit permissions for this resolver directly from the administration panel.
```

Comment:
It might happen that you want to apply.... 

You change is correct, please add "to"